src/hexer/nifcgen.nim

```diff
@@ -633,7 +656,7 @@ proc traverseProc(e: var EContext; c: var Cursor; mode: TraverseMode) =
     inc c
     skip c # skip return type
   else:
-    traverseParams e, c
+    traverseParams e, c, procDecl.pragmas
```

Comment:
But what is in these pragmas that affect pass-by-const-ref?

Reply:
I forgot `byref` is attached to types